apps/calendar/js/views/pan.js

```diff
@@ -37,6 +37,11 @@ function Pan(options) {
   this._onTouchEnd = this._onTouchEnd.bind(this);
   this._tick = this._tick.bind(this);
   this._onTweenEnd = null;
+
+  // RTL
+  var getDir = () => document.documentElement.dir === 'rtl' ? -1 : +1;
+  window.addEventListener('localized', () => this._dir = getDir());
+  this._dir = getDir();
```

Comment:
I think we should abstract this into a separate module.. we are checking for the same thing in too many places. ideally we would have something like `l10nHelper.isRTL` and add the `localized` logic inside the `l10nHelper`.